consensus/polybft/signer/signature.go

```diff
@@ -0,0 +1,81 @@
+package bls
+
+import (
+	"errors"
+
+	bn256 "github.com/umbracle/go-eth-bn256"
+)
+
+type Signature struct {
+	p *bn256.G1
+}
+
+// Verify checks the BLS signature of the message against the public key of its signer
+func (s *Signature) Verify(publicKey *PublicKey, message []byte) bool {
+	hashPoint, err := g1HashToPoint(message)
+	if err != nil {
+		// this should never happen, probably need a log here
```

Comment:
Why not add a logger?

If you can't add a logger here, drop this comment, or return the error